diff -aurN yelp-2.13.5.orig/src/yelp-db-pager.c yelp-2.13.5/src/yelp-db-pager.c
--- yelp-2.13.5.orig/src/yelp-db-pager.c	2006-01-08 21:56:04.000000000 +0100
+++ yelp-2.13.5/src/yelp-db-pager.c	2006-02-13 10:56:05.367251880 +0100
@@ -47,7 +47,7 @@
 #define d(x)
 #endif
 
-#define STYLESHEET_PATH DATADIR"/yelp/xslt/"
+#define STYLESHEET_PATH DATADIR"/sgml/yelp/"
 #define DB_STYLESHEET   STYLESHEET_PATH"db2html.xsl"
 
 #define BOOK_CHUNK_DEPTH 2
diff -aurN yelp-2.13.5.orig/src/yelp-db-print-pager.c yelp-2.13.5/src/yelp-db-print-pager.c
--- yelp-2.13.5.orig/src/yelp-db-print-pager.c	2006-01-14 03:30:26.000000000 +0100
+++ yelp-2.13.5/src/yelp-db-print-pager.c	2006-02-13 10:56:05.371251272 +0100
@@ -47,7 +47,7 @@
 #define d(x)
 #endif
 
-#define STYLESHEET_PATH DATADIR"/yelp/xslt/"
+#define STYLESHEET_PATH DATADIR"/sgml/yelp/"
 #define DB_STYLESHEET   STYLESHEET_PATH"db2html.xsl"
 #define DB_TITLE        STYLESHEET_PATH"db-title.xsl"
 
diff -aurN yelp-2.13.5.orig/src/yelp-info-pager.c yelp-2.13.5/src/yelp-info-pager.c
--- yelp-2.13.5.orig/src/yelp-info-pager.c	2006-01-15 14:59:34.000000000 +0100
+++ yelp-2.13.5/src/yelp-info-pager.c	2006-02-13 10:56:05.389248536 +0100
@@ -41,7 +41,7 @@
 #include "yelp-info-parser.h"
 #include "yelp-settings.h"
 
-#define STYLESHEET_PATH DATADIR"/yelp/xslt/"
+#define STYLESHEET_PATH DATADIR"/sgml/yelp/"
 #define INFO_STYLESHEET STYLESHEET_PATH"info2html.xsl"
 
 #define YELP_INFO_PAGER_GET_PRIVATE(object)(G_TYPE_INSTANCE_GET_PRIVATE ((object), YELP_TYPE_INFO_PAGER, YelpInfoPagerPriv))
diff -aurN yelp-2.13.5.orig/src/yelp-man-pager.c yelp-2.13.5/src/yelp-man-pager.c
--- yelp-2.13.5.orig/src/yelp-man-pager.c	2006-01-29 21:29:30.000000000 +0100
+++ yelp-2.13.5/src/yelp-man-pager.c	2006-02-13 10:56:05.405246104 +0100
@@ -41,7 +41,7 @@
 #include "yelp-man-parser.h"
 #include "yelp-settings.h"
 
-#define STYLESHEET_PATH DATADIR"/yelp/xslt/"
+#define STYLESHEET_PATH DATADIR"/sgml/yelp/"
 #define MAN_STYLESHEET  STYLESHEET_PATH"man2html.xsl"
 
 #define YELP_MAN_PAGER_GET_PRIVATE(object)(G_TYPE_INSTANCE_GET_PRIVATE ((object), YELP_TYPE_MAN_PAGER, YelpManPagerPriv))
diff -aurN yelp-2.13.5.orig/src/yelp-search-pager.c yelp-2.13.5/src/yelp-search-pager.c
--- yelp-2.13.5.orig/src/yelp-search-pager.c	2006-01-28 20:46:37.000000000 +0100
+++ yelp-2.13.5/src/yelp-search-pager.c	2006-02-13 10:56:05.438241088 +0100
@@ -62,7 +62,7 @@
 
 #define YELP_NAMESPACE "http://www.gnome.org/yelp/ns"
 
-#define STYLESHEET_PATH DATADIR"/yelp/xslt/"
+#define STYLESHEET_PATH DATADIR"/sgml/yelp/"
 #define SEARCH_STYLESHEET  STYLESHEET_PATH"search2html.xsl"
 
 typedef gboolean      (*ProcessFunction)        (YelpSearchPager      *pager);
diff -aurN yelp-2.13.5.orig/src/yelp-toc-pager.c yelp-2.13.5/src/yelp-toc-pager.c
--- yelp-2.13.5.orig/src/yelp-toc-pager.c	2006-02-12 06:06:05.000000000 +0100
+++ yelp-2.13.5/src/yelp-toc-pager.c	2006-02-13 10:56:05.455238504 +0100
@@ -60,7 +60,7 @@
 
 #define YELP_NAMESPACE "http://www.gnome.org/yelp/ns"
 
-#define STYLESHEET_PATH DATADIR"/yelp/xslt/"
+#define STYLESHEET_PATH DATADIR"/sgml/yelp/"
 #define TOC_STYLESHEET  STYLESHEET_PATH"toc2html.xsl"
 
 typedef gboolean      (*ProcessFunction)        (YelpTocPager      *pager);
diff -aurN yelp-2.13.5.orig/src/yelp-utils.c yelp-2.13.5/src/yelp-utils.c
--- yelp-2.13.5.orig/src/yelp-utils.c	2006-01-15 15:41:01.000000000 +0100
+++ yelp-2.13.5/src/yelp-utils.c	2006-02-13 10:56:05.463237288 +0100
@@ -156,7 +156,7 @@
 	uri_type = YELP_URI_TYPE_TOC;
     }
     else if (g_str_has_prefix (full_uri, "x-yelp-search:")) {
-	doc_uri = g_strconcat ("file://" DATADIR "/yelp/xslt/search2html.xsl?", full_uri + strlen ("x-yelp-search:"), NULL);
+	doc_uri = g_strconcat ("file://" DATADIR "/sgml/yelp/search2html.xsl?", full_uri + strlen ("x-yelp-search:"), NULL);
 	doc_type = YELP_DOC_TYPE_SEARCH;
 	uri_type = YELP_URI_TYPE_SEARCH;
     }
diff -aurN yelp-2.13.5.orig/stylesheets/Makefile.am yelp-2.13.5/stylesheets/Makefile.am
--- yelp-2.13.5.orig/stylesheets/Makefile.am	2005-10-28 22:37:37.000000000 +0200
+++ yelp-2.13.5/stylesheets/Makefile.am	2006-02-13 10:44:43.596896728 +0100
@@ -1,6 +1,6 @@
 @INTLTOOL_XML_RULE@
 
-xsldir=$(datadir)/yelp/xslt
+xsldir=$(datadir)/sgml/yelp
 
 xsl_DATA =		\
 	db2html.xsl	\
