diff -urN yelp-2.21.90/configure.in yelp-2.21.90.new/configure.in
--- yelp-2.21.90/configure.in	2008-01-28 22:57:59.000000000 +0100
+++ yelp-2.21.90.new/configure.in	2008-02-15 11:59:33.000000000 +0100
@@ -133,7 +133,11 @@
 dnl beagle
 dnl ******
 
-BEAGLE_MODULES="libbeagle-0.0 >= 0.2.4"
+if $PKG_CONFIG --exists libbeagle-1.0; then
+	BEAGLE_MODULES="libbeagle-1.0"
+else
+	BEAGLE_MODULES="libbeagle-0.0 >= 0.2.4"
+fi
 
 AC_ARG_WITH([search],
 	[AC_HELP_STRING([--with-search=basic|beagle|auto],
diff -urN yelp-2.21.90/src/yelp-search-parser.c yelp-2.21.90.new/src/yelp-search-parser.c
--- yelp-2.21.90/src/yelp-search-parser.c	2008-01-28 22:47:22.000000000 +0100
+++ yelp-2.21.90.new/src/yelp-search-parser.c	2008-02-15 11:58:30.000000000 +0100
@@ -562,7 +562,7 @@
 
 	beagle_query_set_max_hits (query, 10000);
 	beagle_query_add_text (query, parser->search_terms);
-	beagle_query_add_source (query, "documentation");
+	beagle_query_add_text (query, "source:documentation");
 
 	parser->hits = g_ptr_array_new ();
 
