diff -urN yelp-2.5.91.orig/configure.in yelp-2.5.91/configure.in
--- yelp-2.5.91.orig/configure.in	2004-03-17 07:43:14.009028384 +0100
+++ yelp-2.5.91/configure.in	2004-03-17 07:43:29.233713880 +0100
@@ -23,7 +23,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE")
 
-ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA eo es et fa fi fr ga gl he hi hr hu id is it ja kn ko li lt lv mk ml mn mr ms ne nl nn no pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA eo es et fa fi fr ga gl he hi hr hu id is it ja kn ko li lt lv mk ml mn mr ms nb ne nl nn pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 # AM_GLIB_GNU_GETTEXT above substs $DATADIRNAME
