diff -ruN yelp-1.0.3/data/images/Makefile.am yelp-1.0.3-/data/images/Makefile.am
--- yelp-1.0.3/data/images/Makefile.am	Tue May 21 03:22:21 2002
+++ yelp-1.0.3-/data/images/Makefile.am	Sat Aug 17 14:20:59 2002
@@ -1,4 +1,4 @@
-imagesdir = $(datadir)/images/yelp
+imagesdir = $(datadir)/pixmaps/yelp
 
 images_DATA = 								\
 	bcg.png								\
diff -ruN yelp-1.0.3/src/Makefile.am yelp-1.0.3-/src/Makefile.am
--- yelp-1.0.3/src/Makefile.am	Thu Jul 11 12:15:00 2002
+++ yelp-1.0.3-/src/Makefile.am	Sat Aug 17 14:24:25 2002
@@ -6,7 +6,7 @@
 	-DSYSCONFDIR=\""$(sysconfdir)"\"				\
 	-DDATADIR=\""$(datadir)"\"					\
 	-DLIBDIR=\""$(libdir)"\"					\
-	-DIMAGEDIR=\""$(datadir)/images/yelp"\"				\
+	-DIMAGEDIR=\""$(datadir)/pixmaps/yelp"\"			\
 	-DGNOMELOCALEDIR=\""$(yelplocaledir)"\"				\
 	-DLIBGNOME_MAN2HTML=\"$(LIBGNOME_MAN2HTML)\"			\
 	-DLIBGNOME_INFO2HTML=\"$(LIBGNOME_INFO2HTML)\"			\
